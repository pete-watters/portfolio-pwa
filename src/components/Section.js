import React, { memo } from 'react';
import PropTypes from 'prop-types';

const Section = ({children}) =>
  <section>
    {children}
  </section>;

Section.propTypes = {
  children: PropTypes.node.isRequired,
};

export default memo(Section);