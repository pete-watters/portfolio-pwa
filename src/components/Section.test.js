import React from 'react';
import { shallow } from 'enzyme';
import Section from './Section';

describe('Section', () => {
  it('should render the component correctly', () => {
    expect(shallow(<Section />)).toMatchSnapshot();
  });
});
