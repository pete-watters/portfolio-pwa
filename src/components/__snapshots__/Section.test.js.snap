// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section should render the component correctly 1`] = `<section />`;
